app/kumactl/data/install/k8s/demo/demo.yaml

```diff
@@ -0,0 +1,90 @@
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: {{ .Namespace }}
+  annotations:
+    kuma.io/sidecar-injection: enabled
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: redis
+  namespace: {{ .Namespace }}
+spec:
+  selector:
+    matchLabels:
+      app: redis
+  replicas: 1
+  template:
+    metadata:
+      labels:
+        app: redis
+    spec:
+      containers:
+        - name: redis
+          image: "redis"
+          ports:
+            - name: tcp
+              containerPort: 6379
+          lifecycle:
+            postStart:
+              exec:
+                command: ["/usr/local/bin/redis-cli", "set", "zone", "{{ .Zone }}"]
+---
+apiVersion: v1
+kind: Service
+metadata:
+  name: redis
+  namespace: {{ .Namespace }}
+spec:
+  selector:
+    app: redis
+  ports:
+  - protocol: TCP
+    port: 6379
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: demo-app
+  namespace: {{ .Namespace }}
+spec:
+  selector:
+    matchLabels:
+      app: demo-app
+  replicas: 1
+  template:
+    metadata:
+      labels:
+        app: demo-app
+    spec:
+      containers:
+        - name: demo-app
+          image: "thefosk/kuma-demo"
+          env:
+            - name: REDIS_HOST
+              value: "redis.kuma-demo.svc.cluster.local"
+            - name: REDIS_PORT
+              value: "6379"
+            - name: APP_VERSION
+              value: "1.0"
+            - name: APP_COLOR
+              value: "#efefef"
+          ports:
+            - name: http
+              containerPort: 5000
+---
+apiVersion: v1
+kind: Service
+metadata:
+  name: demo-app
+  namespace: {{ .Namespace }}
+  annotations:
+    5000.service.kuma.io/protocol: http
+    ingress.kubernetes.io/service-upstream: "true"
```

Comment:
I think I would remove this. Kuma is supposed to add this annotation automatically.